### PR TITLE
logos are epic

### PR DESCRIPTION
**Pull request by Nichodon**
Uh feel free to comment on them and stuff it's very WIP

### CODE REVIEW

File: src/designs/graph.html [hunk 1]
```diff
@@ -6,6 +6,8 @@
     <meta name="description" content="Doo diddly doo I have no internet"/>
     <meta name="viewport" content="width=device-width, initial-scale=1"/>
     <link rel="stylesheet" href="./you-better-rename-this-later-tim.css">
+		
+		<link rel="icon" href="logos/favicon.svg">
```
Comment (SheepTester) [inline src/designs/graph.html:10]: uau, svg favicons, so modern

File: src/designs/logos/draft-2.svg [hunk 2]
```diff
@@ -1,66 +1 @@
...
-    a 10 10 0 0 1 -10 -10
-    v -10
-    a 30 30 0 0 1 30 -30
-    h 10
-    a 10 10 0 0 1 10 10
-    h -20
-    a 20 20 0 0 0 -20 20
-    z
-  "/>
-</svg>
```
Comment (SheepTester) [inline src/designs/logos/draft-2.svg:66]: lol RIP old draft-2

File: src/designs/graph.html (hunk 1 above)
Comment (Nichodon) [reply src/designs/graph.html:10]: i am too lazy to make different sized ico files no thx

File: src/designs/logos/draft-2.svg (hunk 2 above)
Comment (Nichodon) [reply src/designs/logos/draft-2.svg:66]: gone but not forgotten

**Pull request merged**